packages/components/src/tooltip/test/index.js

```diff
@@ -276,5 +276,63 @@ describe( 'Tooltip', () => {
 			// Tooltip won't show, since the mouse has left the anchor
 			expect( screen.queryByText( 'Help text' ) ).not.toBeInTheDocument();
 		} );
+
+		it( 'should render the shortcut display text when a string is passed as the shortcut', async () => {
+			const user = userEvent.setup( {
+				advanceTimers: jest.advanceTimersByTime,
+			} );
+
+			render(
+				<Tooltip text="Help text" shortcut="shortcut text">
+					<button>Hover Me!</button>
+				</Tooltip>
+			);
+
+			const button = screen.getByRole( 'button', { name: 'Hover Me!' } );
+			await user.hover( button );
+
+			// Tooltip with the shortcut text hasn't appeared yet
+			expect(
+				screen.queryByText( 'shortcut text' )
+			).not.toBeInTheDocument();
+
+			act( () => jest.advanceTimersByTime( TOOLTIP_DELAY ) );
```

Comment:
I see this is part of prior art in this test file, however ideally this should be independent of the tooltip implementation. 

Perhaps we could just do something like this:

```
expect(
	await waitFor( () =>
		screen.getByText( 'shortcut text' )
	)
).toBeVisible();

Reply:
Thanks for the advice! Your method seems to have worked.
However, it was converted to `findByXXX` according to eslint.

```
Prefer `findByText` query over using `waitFor` + `getByText`
```

Reply:
Of course, even better 👍